infra/modules/service/autoscaling.tf

```diff
@@ -24,7 +24,7 @@ resource "aws_appautoscaling_policy" "ecs_scale_policy_memory" {
       predefined_metric_type = "ECSServiceAverageMemoryUtilization"
     }
     scale_in_cooldown  = 300
-    scale_out_cooldown = 5
+    scale_out_cooldown = 30
```

Comment:
I'm increasing the scale out numbers so we doesn't reach for 80 containers immediately 😆